src/Paystack.svelte

```diff
@@ -1,8 +1,14 @@
 <script>
+  // IMPORTS
   import { onMount } from "svelte";
+
+  //PROPS
+  export let config = {};
+  // export let customButton='';
+
+  //STATES
```

Comment:
Remove comments:
In svelte, exported variables are props by default and non-exported are states. Also remove the one on the imports

Reply:
Oh my bad, I just did that for sake of clarity sometimes. I easily get things mixed up.